source/administration/command-line-tools.rst

```diff
@@ -989,6 +992,29 @@ mattermost import slack
     .. code-block:: none
 
       ./mattermost import slack myteam slack_export.zip
+      
+mattermost integrity
+--------------------
+
+  Description
+    Command to check database schema integrity as well as referential integrity of channels, command web hooks, posts, schemes, sessions, users, and teams. This process may temporarily affect live system performance, and should be used during off-peak periods. 
+    
+  Format
+    .. code-block:: none
+
+      mattermost integrity {flag} {flag}
+            
+  Example
+    .. code-block:: none
+
+      ./mattermost integrity --confirm --verbose 
+      
+  Options
+    .. code-block:: none
+
+          --confirm   Confirm you really want to run a complete integrity check that may temporarily harm system performance.
```

Comment:
```suggestion
          --confirm   Skip the confirmation message which indicates that the complete integrity check that may temporarily harm system performance. This is not recommended in production environments.
```

Reply:
I've reworded this due to a misunderstanding on my part.

Reply:
Thanks! Just a slight wording adjustment: `that the complete integrity check that may temporarily harm` --> `that the complete integrity check may temporarily harm`